assigment4/assignment4.py

```diff
@@ -0,0 +1,54 @@
+class Tile:
+ 
+    def __init__(self, row, col, g):
+        self.ROW = row
+        self.COL = col
+        self.graph = g
+ 
+    def isSafe(self, i, j, visited):
```

Comment:
Naming-nit: it looks like this function checks for more than safety. Maybe call it `shouldVisit`? A docstring would also be nice to briefly document the method:

Some discussion about docstrings for example here: https://www.python.org/dev/peps/pep-0257/